README.md

```diff
@@ -36,7 +36,7 @@ If you have Chrome already installed you can use the following command to instal
 
 ### Generating a game
 
-TextWorld provide an easy way of generating simple text-based games via the `tw-make` script. For instance,
+TextWorld provides an easy way of generating simple text-based games via the `tw-make` script. For instance,
```

Comment:
This change makes sense.  The rest are just rewording things for little benefit as far as I can see.  I'd merge this by itself though, feel like undoing the rest of the changes?

Reply:
Done!